website/versioned_docs/version-0.0.1/getting-started.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
0.0.1 -> I would go with the latest version for the moment. We do not want to versioning until we are serious about them (post 1.0.0)